{"name":"childExceptionTesting","status":"broken","statusDetails":{"known":false,"muted":false,"flaky":false,"message":"java.lang.NullPointerException","trace":"java.lang.NullPointerException\n\tat JavaPractice.ExceptionTest.childExceptionTesting(ExceptionTest.java:29)\n\tat sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)\n\tat sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)\n\tat sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)\n\tat java.lang.reflect.Method.invoke(Method.java:498)\n\tat org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)\n\tat org.testng.internal.Invoker.invokeMethod(Invoker.java:583)\n\tat org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)\n\tat org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)\n\tat org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)\n\tat org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)\n\tat org.testng.TestRunner.privateRun(TestRunner.java:648)\n\tat org.testng.TestRunner.run(TestRunner.java:505)\n\tat org.testng.SuiteRunner.runTest(SuiteRunner.java:455)\n\tat org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)\n\tat org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)\n\tat org.testng.SuiteRunner.run(SuiteRunner.java:364)\n\tat org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)\n\tat org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)\n\tat org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)\n\tat org.testng.TestNG.runSuitesLocally(TestNG.java:1137)\n\tat org.testng.TestNG.runSuites(TestNG.java:1049)\n\tat org.testng.TestNG.run(TestNG.java:1017)\n\tat com.intellij.rt.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:66)\n\tat com.intellij.rt.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:110)\n"},"stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1590591894407,"stop":1590591894441,"uuid":"0b9c12a9-0c76-483a-adfa-50b7963e31c8","historyId":"4acb98d40706bccf0e9b67f767f250df","fullName":"JavaPractice.ExceptionTest.childExceptionTesting","labels":[{"name":"package","value":"JavaPractice.ExceptionTest"},{"name":"testClass","value":"JavaPractice.ExceptionTest"},{"name":"testMethod","value":"childExceptionTesting"},{"name":"parentSuite","value":"Default Suite"},{"name":"suite","value":"DevPreactice"},{"name":"subSuite","value":"JavaPractice.ExceptionTest"},{"name":"host","value":"apples-MacBook-Pro.local"},{"name":"thread","value":"8635@apples-MacBook-Pro.local.main(1)"},{"name":"framework","value":"testng"},{"name":"language","value":"java"}],"links":[]}